partner_delivery_schedule/views/report_shipping.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="report_picking" inherit_id="stock.report_picking">
+        <xpath expr="//div[@name='div_sched_date']" position="after">
+            <div
+                t-if="o.picking_type_id.code == 'outgoing' and o.partner_id"
+                class="col-auto"
+            >
+                <t
+                    t-set="partner"
+                    t-value="o.partner_id if o.partner_id.type == 'delivery' else o.partner_id.commercial_partner_id"
+                />
+                <strong>Delivery Schedule</strong>
+                <t t-foreach="partner.delivery_schedule_ids" t-as="horary">
+                    <div>
+                        <span
+                            class="badge badge-pill badge-dark"
```

Comment:
I think the substitute classes are rounded-pill bg-dark. 

You can check with this script the xml and save you defining a style file:https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade_160.py#L164-L169

Reply:
Thanks!!